### PR TITLE
Added PWA support

### PR DESCRIPTION
**Pull request by GeoDerp**
# Added Tablet Support
  - Added manifest.json file for tablet.html to enable PWA support (Tablet web app)
  - modified /tablet.html to support the PWA implementation
    - disabled doable tap zoom (IOS & Android)
    - IOS icon support
  - /public/img/brand/  now has different size icons 
   *(designed for PWA, manifest.json and IOS icons from tablet.html )*

- Re-res'd the icon *(5300px)*
  - Tif file [here](https://drive.google.com/drive/folders/1mGp5emhDJwglZ3Cfb2cVc_0ruvM78sAj?usp=sharing): 

## How to add a PWA to the Home Screen  
 **IOS** *via Safari*: on URL (*IP-Address/tablet.html*),
   ```share > add to home screen``` <br>
  **Android** *via chrome*:  on URL (*IP-Address/tablet.html*),
   ```menu (hamburger on the top right) > add to Home screen```

### Icon:
**Android:**
Icon  

**IOS** (Needs square logos):
IOS Icon  

### App interface:
**Android** (image using 12 bank):
App

**IOS** (image using 32 bank)
IOS App

### CODE REVIEW

File: public/tablet.html [hunk 1]
```diff
@@ -16,7 +16,17 @@
...
-	<meta name="viewport" content="">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0 , maximum-scale=1.0, user-scalable=0">
+	<link rel="manifest" href="/manifest.json">
+	<!-- PWA support -->
+	<meta name="theme-color" content="#000000"/>
+	<!-- ios support -->
+	<link rel="apple-touch-icon" href="/img/brand/icons-72-square.png">
+	<link rel="apple-touch-icon" href="/img/brand/icons-96-square.png">
+	<link rel="apple-touch-icon" href="/img/brand/icons-128-square.png">
+	<link rel="apple-touch-icon" href="/img/brand/icons-512-square.png">
```
Comment (MarcHagen) [inline public/tablet.html]: You should add `sizes="512x512"` if you're defining `apple-touch-icon` multiple times

File: public/tablet.html [hunk 2]
```diff
@@ -57,13 +67,22 @@
 			border: 1px solid rgba(255,255,255,0.15);
 		}
 
+		.disable-dbl-tap-zoom {
+  		touch-action: manipulation;
```
Comment (MarcHagen) [inline public/tablet.html]: Tab indent?

Comment (GeoDerp) [reply public/tablet.html]: Done :) 22859378d848d6b0d0a7d4b1ee6238e522469f8a

File: public/tablet.html (hunk 1 above)
Comment (GeoDerp) [reply public/tablet.html]: Done :) 8b3167fd0e294528606d75d2a22f46c05062c123

**Comment by GeoDerp** [comment]
Not sure what's happening on 8b3167fd0e294528606d75d2a22f46c05062c123 with @CLAassistant, hopefully it fixes itself or you can pull all commits without it

**Pull request closed without merging**